src/pages/capabilities.tsx

```diff
@@ -512,21 +512,21 @@ function FeaturesPage() {
               variants={transitions.item}
               className="tw-heading-4 md:tw-heading-3 mb-4 "
             >
-              Chain-key transactions
+              Cross-chain transaction signing
             </motion.h3>
             <motion.p
               className="tw-paragraph md:tw-lead-sm text-black-60 mb-0"
               variants={transitions.item}
             >
-              Chain-key signatures are the innovation that powers direct
-              integration with other blockchains without the use of bridges. It
-              allows the Internet Computer to natively create transactions on
-              other blockchains like Bitcoin and Ethereum. Currently, the ECDSA
-              signature scheme, and with it most other blockchains, are
-              supported.
+              Threshold ECDSA, coupled with chain-key cryptography is the key
+              innovation behind direct integration with other blockchains without
```

Comment:
chain-key cryptography is the main innovation behind the Internet Computer's direct integration with other blockchains,